### PR TITLE
ENG-1490 UX alignment based on recent prototype

### PR DESCRIPTION
**Pull request by jeffgo10**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **joewhite101**

If they are not already assigned, you can assign the PR to them by writing `/assign @joewhite101` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-de-app).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando-k8s/entando-de-app/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/main/webapp/resources/static/css/ootb/content-templates/ct-news.css
Comment (raxkaynan) [inline src/main/webapp/resources/static/css/ootb/content-templates/ct-news.css:99]:
```suggestion
}

```

**Review by raxkaynan — CHANGES_REQUESTED** [review]
@jeffgo10 Looks good. Just one really minor change needed.

**Review by raxkaynan — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando-k8s/entando-de-app/pull/139#pullrequestreview-529936876" title="Approved">raxkaynan</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **joewhite101**

If they are not already assigned, you can assign the PR to them by writing `/assign @joewhite101` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-de-app).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando-k8s/entando-de-app/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
